README.md

```diff
@@ -399,3 +399,34 @@ gui.Stack{
 ```
 
 ![Screenshot](https://user-images.githubusercontent.com/3182651/215946217-3705dbd1-4ec8-4aed-a9eb-381fecb2d8f2.png)
+
+### Set form as player inventory
+
+A form can be set as the player inventory. Flow internally generates the
+formspec and passes it to `player:set_inventory_formspec()`.
```

Comment:
Perhaps the incompatibility with other inventory mods (like sfinv) should be mentioned?

Reply:
Sure. Something like "This will override your inventory mod?"

Reply:
Maybe "This will completely replace your inventory and isn't compatible with inventory mods like sfinv."

Reply:
Oh, right. That's even better.